services/src/handlers/wms.rs

```diff
@@ -316,7 +316,10 @@ async fn wms_map_handler<C: Context>(
             Box::new(irp)
         };
 
-        let processor = initialized.query_processor().context(error::Operator)?;
+        let processor = execution_context
```

Comment:
save_dataset and maybe others (?) are missing

Reply:
is now automatically checked because every operator is wrapped: 10bc0646cd8e6938210e1ff24a11e9124fd983af